profile_grants_list.go

```diff
@@ -0,0 +1,25 @@
+package linodego
+
+import (
+	"context"
+	"log"
+)
+
+type GrantsListResponse = UserGrants
+
+func (c *Client) GrantsList(ctx context.Context) (*GrantsListResponse, error) {
+	e := "profile/grants"
+	r, err := coupleAPIErrors(c.R(ctx).SetResult(GrantsListResponse{}).Get(e))
+	if err != nil {
+		return nil, err
+	}
+
+	if r.StatusCode() == 204 {
+		log.Printf(
```

Comment:
Hmm, wonder if we should add a `SetWarnings(enabled)` function to `Client` to enable and disable warnings like these

Reply:
Yea, that would be very helpful. Just realized that for this PR, actually, this warning is never reached. I am going to remove it. I will gather the warning/info log as well as 204 handling into another utility function in another PR.